Translation corrections (#334)
07th-mod/higurashi-console-arcs: Update/kake_077.txt

//r…それとも甘えるな、と突き放すべきなのか。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "…それとも甘えるな、と突き放すべきなのか。",

> 			NULL, "...Or should scold him for taking advantage of my position?", Line_Normal);

Reviewer: ```suggestion
			NULL, "...Or should I scold him for taking advantage of my position?", Line_Normal);
```

			NULL, "\"Why were you drinking all the way out here in the first place...?", Line_WaitForInput);
	ModPlayVoiceLS(4, 28, "ps3/s20/28/vtk2_tomoe2175", 256, TRUE);
	OutputLine(NULL, "　接待慣れして俺は酒に強い、って前に豪語されていたくせに。",
			NULL, " You bragged before about being good at wining and dining, and a strong drinker. ", GetGlobalFlag(GLinemodeSp));
	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#9d7f69>巴</color>", NULL, "<color=#9d7f69>Tomoe</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 28, "ps3/s20/28/vtk2_tomoe2176", 256, TRUE);
	OutputLine(NULL, "飲み過ぎるといろんな人に迷惑かかるって仰っていたのは、前野さんですよ？」",

> 			NULL, "Yet here you are, getting excessively drunk and bothering others, Maeno-san.\"", Line_Normal);

Reviewer: ```suggestion
			NULL, "Yet here you are, getting excessively drunk and being a nuisance, Maeno-san.\"", Line_Normal);
```
